corefxlab.sln

```diff
@@ -123,6 +123,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ALCProxy.TestInterfaceUpdat
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ALCProxy.TestInterface", "tests\ALCProxy.TestInterface\ALCProxy.TestInterface.csproj", "{6435C875-F526-45E1-97E1-B5BF8C4F9A55}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Text.CaseFolding", "src\System.Text.CaseFolding\System.Text.CaseFolding.csproj", "{6B8DAAD7-6CC3-4138-B719-02AE81BEC535}"
```

Comment:
Why does BDN or VS care about this particular project? Are you or anyone else within the solution depending on it?

cc @iSazonov, @tarekgh - do you know if this was intentionally left out of the solution file or was it an oversight?

Reply:
I don’t know how to integrate the new code (I mean my SimpleCaseFolding) into this repo in right way. If I remember correctly, I asked maintainers about this and the answer was that nothing was needed except merge my PR as is.

Reply:
BDN cares because our Benchmarks project has a reference to it:

https://github.com/dotnet/corefxlab/blob/5c25321148e3ccfd2a40d52e3286abda829741e4/tests/Benchmarks/Benchmarks.csproj#L55-L57

Reply:
Makes sense. Looks good to me.